src/gsi/setuprad.f90

```diff
@@ -1456,6 +1461,7 @@ subroutine setuprad(obsLL,odiagLL,lunin,mype,aivals,stats,nchanl,nreal,nobs,&
            do i=1,nchanl
               m=ich(i)
               if (icld_det(m)>0 .and. varinv(i) >= tiny_r_kind) then
+!             if (icld_det(m)>0 .or. varinv(i) >= tiny_r_kind) then
```

Comment:
Why did you add this comment?

Reply:
I will delete this comment.  I thought there was an issue with the code, so I added this commented line.  I was wrong.  It was correct with the .and..  

Thank you for looking at it closely.